07-modeling.Rmd

```diff
@@ -391,9 +396,9 @@ where $y_i$ is the outcome, $\beta_0$ is an intercept, and $x_1, \cdots, x_n$ ar
 
 Assumptions in logistic regression using survey data include:
 
-- The outcome variable has two levels
-- There is a linear relationship between the independent variables and the log odds \@ref(eq:logoddlin)
-- The residuals are homoscedastic, that is, the error term is the same across all values of independent variables
+  - The outcome variable has two levels
+  - There is a linear relationship between the independent variables and the log odds \@ref(eq:logoddlin) 
```

Comment:
This just shows up in the book as (7.1).  At first I thought it was a section, but seeing the code I realize it is an equation.  Can we add something to make it clear that we're referencing this as an equation and not a section?